### ISSUE TITLE
Add a menuSelect widget

### ISSUE THREAD

**Issue by vvo** [algolia/instantsearch.js#1904]
**Do you want to request a *feature* or report a *bug*?**
feature

**Feature: What is your use case for such a feature?**
Redering a Menu as a select has been asked too many times to be ignored anymore. We need a menuSelect no matter how far we go in providing advanced rendering customisation.

**Feature: What is your proposed API entry? The new option to add? What is the behavior?**

```js
search.addWidget(instantsearch.widgets.menuSelect({
  attributeName: '...',
  limit: ...
  label: '...'
}));
```

This should render:

```html
<label>... <select><option></option></select></label>
```

Ref: https://github.com/algolia/instantsearch.js/issues/868

**What project are you opening an issue for?**
- instantsearch.js

**What is the version you are using? Always use the latest one before opening a bug issue.**
latest

**Comment by combs** [comment]
thank you! 
my comment from another related issue--
> this would be very nice as a basic builtin for instantsearch.js. Would make Algolia instantsearch.js a lot more mobile friendly, which would of course help the bulk of our 1m+ monthly unique users.

**Comment by bobylito** [comment]
Now that v2 has connectors this should be fairly trivial to implement with `connectMenu`. Let's move forward with it based on the options of `menu`.

**Comment by Haroenv** [comment]
This is an example of `connectMenu` 

https://github.com/algolia/instantsearch.js/blob/develop/dev/app/custom-widgets/vanilla/selectMenu.js

**Comment by bobylito** [comment]
This has been released in [2.2](https://github.com/algolia/instantsearch.js/blob/develop/CHANGELOG.md#features) ([see the doc](https://community.algolia.com/instantsearch.js/v2/widgets/menuSelect.html))
